src/main/java/org/jabref/gui/maintable/MainTableNameFormatter.java

```diff
@@ -37,7 +37,7 @@ public String formatNameLatexFree(final String nameToFormat) {
 
         switch (nameFormatPreferences.getDisplayStyle()) {
             case AS_IS:
-                return nameToFormat;
+                return authors.getAsUnformattedLatexFree();
```

Comment:
Your fix works but needs quite some changes. What about adding a test here https://github.com/JabRef/jabref/blob/41678ce94308cb39849d23fb2217f11f2d7c2b84/src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java#L127
that makes sure that the complicated author parsing code is only invoked if the display style is not `AS_IS`? Should also do the trick, right?

Reply:
Checking `AS_IS` in 
> 
> https://github.com/JabRef/jabref/blob/41678ce94308cb39849d23fb2217f11f2d7c2b84/src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java#L127
> 
> 
will indeed produce less code (and frankly, I didn't even think about that change). My argument for the current structure is to,
1. let `MainTableNameFormatter` handle the user's preferences,
2. let `AuthorList` handle all formatting/caching for authors.

I guess I am a proponent of [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) 😛 

Anyway, I have updated with an attempt at the described changes!